embedding_matrix_api/app/schemas/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""Schemas __init__."""
+
+from .model import (
+    ModelInputData,
```

Comment:
@vicente87 Why do you use `ModelInputData` in this service?

Reply:
Here we do not use it but as I told you before that we had taken the pattern of services that you gave us and we did not modify many things and that is why they may not be used